develop.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+# One-time run:
+# Build the docker image.
+# Comment after running once.
+# It takes a while.
+# ./dretesteth.sh build
+
+# Now dretesteth.sh should act like retesteth.
+
+# FIXME: set to fit your local environment. Where are the ethereum/tests? 
+testspath=/home/ia/dev/ethereum/tests
+
+# Run a single test.
+# ./dretesteth.sh -t GeneralStateTests/stArgsZeroOneBalance -- --testpath "${testspath}" --clients t8ntool --singletest addmodNonConst
+
+# Run a single test, alternate command.
+# NOTE: It is not clear what the root directory of the 'testfile' argument should be. "tests/"? The absolute path to the tests/ directory?
+#       The former works for me as expected, the latter seems to cause ./dretesteth to run eternally (or a long time). 
```

Comment:
Ths is a bug

The path to file should be as of inside the docker where testpath is mounted

Reply:
absolute